codazen/webDiplomacy
# Feature/fbfw 109 draw units on gameboard map

### Summary
This PR adds a function to draw units to the map. At the moment the function is called whenever the beta app loads. The PR also includes some interface changes and new data structures to support this and future map updates. 

This PR is branched off of https://github.com/codazen/webDiplomacy/pull/71

### Video
https://drive.google.com/file/d/1IouI0Aq49SAwTv30Bp-MBbN00cV-hJgB/view?usp=sharing

### Testing
1. Spool up local host environment
2. Go to an existing / create new game
3. Click on "Play Beta"
4. Verify unit placement is correct on beta map
5. Proceed to a new turn
6. Reload beta map and verify correct unit placement again
